kymatio/frontend/torch_f.py

```diff
@@ -0,0 +1,23 @@
+import torch.nn as nn
+
+class Scattering(nn.Module):
+   def __init__(self, J, shape, max_order = None):
+       super(Scattering, self).__init__()
+       return
+       self.J = J
+       self.shape = shape
+       self.max_order = max_order
+       self.build()
+
+   def build(self):
+       return
```

Comment:
remove - this will not error even in the current state

Reply:
The idea is that this function should be erased. In C++ there is an instruction(virtual AFAIR) to say it should be overloead